tests/generators/VegetaLoadGenerator.ts

```diff
@@ -42,12 +43,11 @@ export default class VegetaLoadGenerator {
       fs.writeFileSync(absoluteFolderPath + `/requests/create${i}.json`, createOperationBuffer);
 
       // Compute the DID unique suffix from the generated Create payload.
-      const didUniqueSuffix = Did.getUniqueSuffixFromEncodeDidDocument(createPayload, ProtocolParameters.get(1000000).hashAlgorithmInMultihashCode);
+      const didUniqueSuffix = Did.getUniqueSuffixFromEncodeDidDocument(createPayload, ProtocolParameters.get(blockchainTime).hashAlgorithmInMultihashCode);
```

Comment:
I don't see tests in this PR for this, but I see there is an open issue for handling it later: https://github.com/decentralized-identity/sidetree/issues/265

Reply:
#265 is for something else, this change is more generic, it is simply removing a hardcoded value.

I think what you are looking for is a way to run load test using this tool, which is tracked by #119, though I am still guity for not doing it because it is not consider "mainnet" blocking.

Reply:
yes, basically want to understand how to implement protocol upgrades, and have unit / integration tests around that.